cups/raster-interpret.c

```diff
@@ -712,26 +712,26 @@ cleartomark_stack(_cups_ps_stack_t *st)	/* I - Stack */
 
 static int				/* O - 0 on success, -1 on error */
 copy_stack(_cups_ps_stack_t *st,	/* I - Stack */
-           int              c)		/* I - Number of objects to copy */
+           int              count)		/* I - Number of objects to copy */
```

Comment:
Fix comment indentation, although I'm not sure it is worthwhile to change this code that is going away.